tests/functional-tests/README.md

```diff
@@ -16,62 +13,60 @@ This project contains an automation framework to exercise the IBM Spectrum Scale
 
 Install Python (3.7.4 or higher) and below mentioned pip modules:
 
-  ``` 
-        python3.7 -m pip install kubernetes
-        python3.7 -m pip install pytest
-        python3.7 -m pip install pytest-html
-        python3.7 -m pip install jsmin
-  ```
-       
+``` 
+python3.7 -m pip install kubernetes
+python3.7 -m pip install pytest
+python3.7 -m pip install pytest-html
+python3.7 -m pip install jsmin
+```
 
 ### How to run IBM Spectrum Scale CSI test automation
-- Clone the source code from [git](https://github.com/IBM/ibm-spectrum-scale-csi/) repository.
-
-- Configure parameters in [tests/functional-tests/config.json](https://github.com/IBM/ibm-spectrum-scale-csi/blob/dev/tests/functional-tests/config.json) file. 
-
-- Set node labels for "attacherNodeSelector","provisionerNodeSelector","pluginNodeSelector" in config.json file
-
-
-- cluster-config parameter is mandatory to pass the SpectrumScale & CSI Operator/Driver images configuration as input.
 
-                For example :
-                   pytest  driver_test.py --clusterconfig config.json
-                   
-- If kubeconfig file is at not at ~/.kube/config location, pass the correct location with --kubeconfig (optional)               
-                
-                For example :
-                   pytest  driver_test.py --clusterconfig config.json --kubeconfig <kubeconfig_file_path>
+- Configure parameters in [config.json](./tests/functional-test/config.json) file.
+
+- Define kubernetes/Openshift Spectrum Scale node labels for `attacherNodeSelector`,`provisionerNodeSelector`,`pluginNodeSelector` in config.json file
+
+- `--clusterconfig` parameter is mandatory to pass IBM Spectrum Scale API Credentials & CSI Operator/Driver image specific configuration as input.
+```
+For example :
+  pytest  driver_test.py --clusterconfig config.json
+```                   
+- If kubeconfig file is at not at `~/.kube/config` location, pass the correct location with `--kubeconfig` (optional)
+```
+For example :
+  pytest  driver_test.py --clusterconfig config.json --kubeconfig <kubeconfig_file_path>
+```
+- For generating the report in html format, pass the `--html` (optional)
+```
+For example :
+  pytest  driver_test.py --clusterconfig config.json --html report.html
+```
 
-- For generating the report in html format, pass the --html (optional)               
-                
-                For example :
-                   pytest  driver_test.py --clusterconfig config.json --html report.html
-                   
-                
 ### Run driver tests using driver_test.py as shown below -
-       
-                cd csi-automation/scale_operator
+```
+cd ./tests/functional-tests/
```

Comment:
another nit-picky... This line and line 60 are consistent, should drop the end `/` or add the`/` to line 60